SU-SWS/saa_victoria_subtheme
# Adapt 476  small horizontal card alignment

# READY FOR REVIEW

# Summary
- set padding top = 0 for h2 and h3 in the small horizontal layout.

# Review By (Date)
- 7/20
# Urgency
- important before tomorrow, small noticeable alignment

# Steps to Test

1. pull down branch
1. clear the cache
2. check that h2 and h3 align WITH and WITHOUT subtitle text above them in the SMALL horizontal cards (circle or square)

# Affected Projects or Products
- n/a

# Associated Issues and/or People
- JIRA ticket - DAPT-476 testing
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
